projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -328,6 +328,20 @@ public function get_upsell_nudge() {
 		}
 	}
 
+	/**
+	 * Adds Jetpack menu.
+	 *
+	 * @param int  $position The position of the menu item in the menu.
+	 * @param bool $seperator Whether to add a separator before the menu item.
+	 */
+	public function add_jetpack_menu( $position = 2, $seperator = false ) {
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+			parent::add_jetpack_menu( $position, $seperator );
+		} else {
+			parent::add_jetpack_menu();
```

Comment:
Will need to change this line to `parent::add_jetpack_menu( $position, $seperator );` to see the changes mentioned in the PR description.